unstable/wlr-surface-layers-unstable-v1.xml

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<protocol name="wlr_surface_layers_unstable_v1">
+  <copyright>
+    Copyright © 2017 Drew DeVault
+
+    Permission to use, copy, modify, distribute, and sell this
+    software and its documentation for any purpose is hereby granted
+    without fee, provided that the above copyright notice appear in
+    all copies and that both that copyright notice and this permission
+    notice appear in supporting documentation, and that the name of
+    the copyright holders not be used in advertising or publicity
+    pertaining to distribution of the software without specific,
+    written prior permission.  The copyright holders make no
+    representations about the suitability of this software for any
+    purpose.  It is provided "as is" without express or implied
+    warranty.
+
+    THE COPYRIGHT HOLDERS DISCLAIM ALL WARRANTIES WITH REGARD TO THIS
+    SOFTWARE, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY AND
+    FITNESS, IN NO EVENT SHALL THE COPYRIGHT HOLDERS BE LIABLE FOR ANY
+    SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN
+    AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+    ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF
+    THIS SOFTWARE.
+  </copyright>
+
+  <interface name="zwlr_surface_layers_v1" version="1">
```

Comment:
I've brought this up before but we never came to a satisfying answer.

When I first saw this name of this interface, I was really confused as to what it does. This interface (`zwlr_surface_layers_v1`) is a shell. All the other shells have "shell" somewhere in their name and I think there is a good reason to continue this.

Here are my suggestions:

* zwlr_layer_surface_shell_v1
* zwlr_surface_layers_shell_v1

Reply:
Sounds reasonable to me

Reply:
I disagree with `zwlr_layer_surface_shell_v1`. layer-surface isn't a shell.

Reply:
The layer surface isn't a shell, but the thing creating layer surfaces is.